site/_js/fugu-showcase.js

```diff
@@ -86,6 +88,102 @@ import './web-components/enhanced-select';
     new URL(document.location.href).searchParams.get('api')?.split(',') || [];
   onSearch();
 
+  if ('share' in navigator && 'canShare' in navigator) {
+    const imageToPNG = async blob => {
```

Comment:
Gave this a quick test on iOS. While I like using the capabilities of the Web Share API I think I'd vote to remove this. WhatsApp seems to ignore the image anyway, iMessage takes it but this introduces weirdness for users as the links and the image would appear as distinct entries in the "Shared with you entry". Also usually the app sites have their own og:image set anyway, which would probably be a better fit than our own screenshot too.

Reply:
It works super well on Twitter, which is our primary share target I guess. I vouch for keeping it.

Reply:
Well, looks like I was biased when testing 😄 sounds good to me, since you could manually edit what's shared on iMessage.